code/modules/surgery/mcomp_tendwounds.dm

```diff
@@ -0,0 +1,157 @@
+/datum/surgery/mcomp_wounds
+	name = "Tend Damage (Brute/Burn)"
+	possible_locs = ALL_LIMBS
+	invasiveness = list(SURGERY_DEPTH_SURFACE)
+	required_surgery_skill = SKILL_SURGERY_TRAINED
+	var/required_trait = TRAIT_YAUTJA_TECH// Only predators can do this
+
+	pain_reduction_required = 0
+	steps = list(
+		/datum/surgery_step/mstabilize_wounds,
+		/datum/surgery_step/mtend_wounds,
+		/datum/surgery_step/cauterize/mclamp_wound,
+				)
+	lying_required = FALSE
+	self_operable = TRUE
+
+/datum/surgery/mcomp_wounds/can_start(mob/living/carbon/human/user, mob/living/carbon/human/patient, var/obj/limb/L, obj/item/tool)
+	if((istype(user) && HAS_TRAIT(user, TRAIT_YAUTJA_TECH)) && (patient.getBruteLoss() || patient.getFireLoss()))
```

Comment:
if they have no damage you should display an error message explaining that they have no damage and that's why you can't start the surgery

may be confusing otherwise